app/controllers/services/service_bindings_controller.rb

```diff
@@ -22,11 +22,16 @@ class ServiceBindingsController < RestController::ModelController
     get path_guid, :read
 
     def read(guid)
-      obj = find_guid(guid)
-      raise CloudController::Errors::ApiError.new_from_details('ServiceBindingNotFound', guid) if obj.v2_app.blank?
+      service_binding = find_guid(guid)
+      raise CloudController::Errors::ApiError.new_from_details('ServiceBindingNotFound', guid) if service_binding.v2_app.blank?
```

Comment:
Little surprised that changes were made to v2 api since we've generally avoided making changes to it.  That said changes seem fine

Reply:
We still have some users using v2 and thus it makes sense to do this in v2 as well